ds3-sdk/src/main/java/com/spectralogic/ds3client/helpers/events/ConcurrentEventRunner.java

```diff
@@ -0,0 +1,28 @@
+/*
+ * ******************************************************************************
+ *   Copyright 2014-2016 Spectra Logic Corporation. All Rights Reserved.
+ *   Licensed under the Apache License, Version 2.0 (the "License"). You may not use
+ *   this file except in compliance with the License. A copy of the License is located at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *   or in the "license" file accompanying this file.
+ *   This file is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR
+ *   CONDITIONS OF ANY KIND, either express or implied. See the License for the
+ *   specific language governing permissions and limitations under the License.
+ * ****************************************************************************
+ */
+
+package com.spectralogic.ds3client.helpers.events;
+
+import java.util.concurrent.Executor;
+import java.util.concurrent.Executors;
+
+public class ConcurrentEventRunner implements EventRunner {
+
+    private final static Executor EVENTS_EXECUTOR = Executors.newFixedThreadPool(4);
```

Comment:
Not specifically, it seemed like a reasonable value since events should be short lived, and we don't want to have too many threads created.